ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-release-1.17.yaml

```diff
@@ -0,0 +1,773 @@
+    name: hyperconverged-cluster-bundle
+    namespace: ci
+    tag: 1.17.0-unstable
+  hco-bundle-reg-prev:
+    name: hyperconverged-cluster-bundle
+    namespace: ci
+    tag: 1.16.0-unstable
+  ocp_builder_rhel-9-golang-1.25-openshift-4.21:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.25-openshift-4.21
+binary_build_commands: make install
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-9-release-golang-1.25-openshift-4.21
+images:
+- dockerfile_path: build/Dockerfile.okd
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/hyperconverged-cluster-operator
+  to: hyperconverged-cluster-operator
+- dockerfile_path: build/Dockerfile.wh.okd
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/hyperconverged-cluster-webhook
+  to: hyperconverged-cluster-webhook
+- dockerfile_path: build/Dockerfile.artifacts
+  optional: true
+  to: virt-artifacts-server
+- dockerfile_path: build/Dockerfile.functest.ci
+  inputs:
+    ocp_builder_rhel-9-golang-1.25-openshift-4.21:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.25-openshift-4.21
+  to: hyperconverged-cluster-functest
+- dockerfile_literal: |
+    FROM src
+    COPY oc /usr/bin/oc
+    RUN ln -s /usr/bin/oc /usr/bin/kubectl
+    RUN dnf install -y jq && dnf clean all
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: hco-oc-bin-image
+- dockerfile_literal: |
+    FROM src
+    COPY oc /usr/bin/oc
+    COPY operator-sdk /usr/local/bin/operator-sdk
+    RUN ln -s /usr/bin/oc /usr/bin/kubectl
+    RUN dnf install -y jq && dnf clean all
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    cli-operator-sdk:
+      paths:
+      - destination_dir: .
+        source_path: /usr/local/bin/operator-sdk
+  to: hco-oc-bin-operator-sdk-image
+- dockerfile_literal: |
+    FROM src
+    COPY oc /usr/bin/oc
+    RUN ln -s /usr/bin/oc /usr/bin/kubectl
+    RUN dnf install -y graphviz jq && dnf clean all
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: hco-oc-bin-image-nightly
+operator:
+  bundles:
+  - as: hco-bundle
+    context_dir: deploy/index-image
+    dockerfile_path: bundle.Dockerfile
+  - as: hco-upgrade-operator-sdk-bundle
+    context_dir: deploy/index-image
+    dockerfile_path: Dockerfile.bundle.ci-index-image-upgrade
+  substitutions:
+  - pullspec: +IMAGE_TO_REPLACE+
+    with: hyperconverged-cluster-operator
+  - pullspec: +WEBHOOK_IMAGE_TO_REPLACE+
+    with: hyperconverged-cluster-webhook
+  - pullspec: +ARTIFACTS_SERVER_IMAGE_TO_REPLACE+
+    with: virt-artifacts-server
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.21"
```

Comment:
why can't we use OCP 4.22 nightly, the actual OCP that this HCO version will be delivered at?
4.22 nighlty builds are available:
https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/#4.22.0-0.nightly